package.json

```diff
@@ -11,6 +11,14 @@
     "url": "https://github.com/MetaMask/metamask-module-template.git"
   },
   "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./package.json": "./package.json"
```

Comment:
This is useful for resolving the root of the package. I'm not sure if it should be added to the module template, but it doesn't hurt I suppose?

We use this in the Snaps repo several times, which doesn't work without this line:

```js
path.dirname(require.resolve('package'));
```

Reply:
Interesting. I guess it doesn't hurt :shrug: